pyproject.toml

```diff
@@ -52,7 +52,7 @@ requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.black]
-line-length = 100
+line-length = 200
```

Comment:
The code style mentions line length of 100 chars, you're changing to 200, I personally think 200 is too much, 100 to 150 its a reasonable line length to allow descriptive code while keeping it readable.  either way this should also be updated: https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/blob/main/docs/contributing/coding-standard.md#line-length

This also should be a separate `build` commit isolated from the actual code changes.

Reply:
I removed this change, in other PR I'll push the change of 100 lines also fixing the lint.sh script and the files that need to be modified by that